client/rest-high-level/src/main/java/org/elasticsearch/client/rollup/GetRollupCapsResponse.java

```diff
@@ -39,7 +39,7 @@ public GetRollupCapsResponse(final Map<String, RollableIndexCaps> jobs) {
     }
 
     public static GetRollupCapsResponse fromXContent(final XContentParser parser) throws IOException {
-        Map<String, RollableIndexCaps> jobs = new HashMap<>();
+        Map<String, RollableIndexCaps> jobs = new TreeMap<>();
```

Comment:
Why does this now care about insert order?

Reply:
i did this on accident, good catch. it is a test sorting order issue iirc that i need to fix.